test/controllers/update/update.beneficial.owner.type.controller.spec.ts

```diff
@@ -71,6 +98,8 @@ describe("BENEFICIAL OWNER TYPE controller", () => {
       expect(resp.text).toContain(BENEFICIAL_OWNER_TYPE_PAGE_GOVERNMENT_BO);
       expect(resp.text).toContain(BENEFICIAL_OWNER_TYPE_PAGE_INDIVIDUAL_MO);
       expect(resp.text).toContain(BENEFICIAL_OWNER_TYPE_PAGE_CORPORATE_MO);
+      expect(mockGetCompanyPscService).not.toHaveBeenCalled();
+      expect(mockSetApplicationData).not.toHaveBeenCalled();
```

Comment:
Won't this change? To not set AppData with new BO data - may set appData for other reasons in future but not for public data retrieval.

Reply:
Sorry I'm not sure I understand, this test is testing that if the `BeneficialOwnerIndividualKey` already exists in the appData, so if a user has already been on this page and API already called, then we check that getCompanyPsc and setApplicationData is not called again.
This current page doesn't currently setApplicationData for anything else. If it does in the future then the test expect can be removed at that point

Reply:
Yes, will fail in future and we will have to remove line 102. Why not remove that now? And just test for not calling REST api again.

Reply:
But ok to do that later

Reply:
Ok, checking mockGetCompanyPscService is not called is enough to cover current functionality so I've removed that setApplicationData check